packages/provider/src/GSNConfigurator.ts

```diff
@@ -37,7 +37,7 @@ export const defaultGsnConfig: GSNConfig = {
   sliceSize: 3,
   relayTimeoutGrace: DEFAULT_RELAY_TIMEOUT_GRACE_SEC,
   methodSuffix: '',
-  requiredVersionRange: '2.2.0',
+  requiredVersionRange: '2.2.3',
```

Comment:
this is the one version we should not update: you basically say that you don't want to run against contracts with older version.